instrumentation/armeria-1.3/library/src/main/java/io/opentelemetry/instrumentation/armeria/v1_3/OpenTelemetryService.java

```diff
@@ -38,7 +38,7 @@ public HttpResponse serve(ServiceRequestContext ctx, HttpRequest req) throws Exc
           .whenComplete()
           .thenAccept(
               log -> {
-                if (HttpStatus.NOT_FOUND.equals(log.responseHeaders().status())) {
+                if (log.responseHeaders().status().equals(HttpStatus.NOT_FOUND)) {
```

Comment:
This change is strange. Was there some warning about the order of comparison?

Reply:
merge from master, this change no longer exists in this PR